### ISSUE TITLE
Task:  Sprint 6 Retrospective, Sprint 7 Planning

### ISSUE THREAD

**Issue by DeeMcCart** [DeeMcCart/CI_PP4_Financial_Planner#33]
EPIC:  <Agile>

### Assumptions:
* Following agile methodology for this project
* Closing out (weekly) sprint should be done with a retrospective 
* Planning of next sprint should be done with backlog grooming and deciding which tasks to move to 'to do' for this sprint

### Acceptance Criteria
* Sprint6 closed off (this happens automatically when using iterations) 
* Any issues which were linked to Sprint6 but not in 'Done' status reviewed and
- moved to next sprint for completion of outstanding tasks
- OR if the acceptance criteria is met and tasks done, but there are issues, then log the issues as a bug and closeout the original UserStory/ task
- got to respect timeboxing and not try to complete any tiny outstanding items from previous Sprint in Sprint 7.  The time has passed so now deal with any hangover items in CURRENT sprint

### Tasks
- [x] Task1 Retrospective
- [x] Task2 Backlog grooming
- [x] Task3 Sprint Planning and issue prioritisation

**Comment by DeeMcCart** [comment]
Hmmm I was  bit concerned that, despite putting in a lot of hours, I had not achieved what I had hoped for in Sprint 6 and was required to return a number of issues into the backlog.  However, I created burndown charts in github and realised that I had actually completed a lot of work during Sprint6.  

First is the burndown chart by EPIC - This shows that I completed 13 SPs of MVP work, and 30 SPs towards Release1 'Personal Financial Plan' .  I also spent time on UX, Bugs and Agile.   Total 55 SPs.  Not all of this work was 100% Sprint6 as some of the MVP issues already had work done against them in previous sprints but its enough to say they were completed in Sprint6.   Viewing by EPIC allows me to broadly see what themes I was working against.


![Image](https://github.com/DeeMcCart/CI_PP4_Financial_Planner/assets/126502176/7187705f-5852-4a0d-9167-1ac876d6ba1a)




And here is the burndown chart by Label - this is useful as it shows that I am moving from the highest-priority items (Must-Have: 7 SPs) and starting to progress towards the next priority (Should-Have: 38 SPs, Could-Have 6 SPs).  It also shows 7 SPs spent on bug resolution, this is slightly higher than the SPs (3) for EPIC bugs on the previous chart, that is because some of the items labelled 'bug' were associated with their primary EPIC, e.g. UX.  I think that's grand and its a sign the classification/measurement system is working as I'd hoped.
 


![Image](https://github.com/DeeMcCart/CI_PP4_Financial_Planner/assets/126502176/7639a8a7-cce8-49b5-96a3-0b1a44492d5a)
